### PR TITLE
fix dockerfile build

### PR DESCRIPTION
**Pull request by Evanfeenstra**
Adding `mako` and `mrkd` fixes `docker build` for us

### CODE REVIEW

**Comment by cdecker** [comment]
Good catch, however I think the apropriate place to add these would be `pyproject.toml` and `poetry.lock` so all environments get them?

**Comment by niftynei** [comment]
We removed the `mrkd` and `mistune` dependencies in master, seems wrong to be adding them back?

File: Dockerfile [hunk 1]
```diff
@@ -108,6 +108,8 @@ RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/inst
     && pip3 install -U wheel \
     && /root/.local/bin/poetry config virtualenvs.create false \
```
Comment (niftynei) [inline Dockerfile:109]: should we just convert this over to building a requirements.txt and installing w/ pip as well, instead of using `poetry install`?

Comment (cdecker) [reply Dockerfile:109]: Probably

**Comment by cdecker** [comment]
> We removed the `mrkd` and `mistune` dependencies in master, seems wrong to be adding them back?

Right, in that case we should likely ensure that `mako` and `mrkd` are not needed to build at all, and we need to ensure that all generated files are checked in correctly.

**Comment by Evanfeenstra** [comment]
thanks for looking into it

**Pull request closed without merging**